src/System.Diagnostics.DiagnosticSource/dir.props

```diff
@@ -4,5 +4,7 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.2.0</AssemblyVersion>
     <IsNETCoreApp>true</IsNETCoreApp>
+    <IsUAP>true</IsUAP>
+    <IsUAPRef>false</IsUAPRef>
```

Comment:
We will eventually want to revisit these IsUAPRef=false choices because we will likely just include them as refs as well.